cross/libpam/PLIST

```diff
@@ -0,0 +1,187 @@
+share/locale/cy/LC_MESSAGES/Linux-PAM.mo
+share/locale/da/LC_MESSAGES/Linux-PAM.mo
+share/locale/de/LC_MESSAGES/Linux-PAM.mo
+share/locale/de_CH/LC_MESSAGES/Linux-PAM.mo
+share/locale/el/LC_MESSAGES/Linux-PAM.mo
+share/locale/eo/LC_MESSAGES/Linux-PAM.mo
+share/locale/es/LC_MESSAGES/Linux-PAM.mo
+share/locale/et/LC_MESSAGES/Linux-PAM.mo
+share/locale/eu/LC_MESSAGES/Linux-PAM.mo
+share/locale/fa/LC_MESSAGES/Linux-PAM.mo
+share/locale/fi/LC_MESSAGES/Linux-PAM.mo
+share/locale/fr/LC_MESSAGES/Linux-PAM.mo
+share/locale/ga/LC_MESSAGES/Linux-PAM.mo
+share/locale/gl/LC_MESSAGES/Linux-PAM.mo
+share/locale/gu/LC_MESSAGES/Linux-PAM.mo
+share/locale/he/LC_MESSAGES/Linux-PAM.mo
+share/locale/hi/LC_MESSAGES/Linux-PAM.mo
+share/locale/hr/LC_MESSAGES/Linux-PAM.mo
+share/locale/hu/LC_MESSAGES/Linux-PAM.mo
+share/locale/ia/LC_MESSAGES/Linux-PAM.mo
+share/locale/id/LC_MESSAGES/Linux-PAM.mo
+share/locale/is/LC_MESSAGES/Linux-PAM.mo
+share/locale/it/LC_MESSAGES/Linux-PAM.mo
+share/locale/ja/LC_MESSAGES/Linux-PAM.mo
+share/locale/ka/LC_MESSAGES/Linux-PAM.mo
+share/locale/kk/LC_MESSAGES/Linux-PAM.mo
+share/locale/km/LC_MESSAGES/Linux-PAM.mo
+share/locale/kn/LC_MESSAGES/Linux-PAM.mo
+share/locale/ko/LC_MESSAGES/Linux-PAM.mo
+share/locale/kw_GB/LC_MESSAGES/Linux-PAM.mo
+share/locale/ky/LC_MESSAGES/Linux-PAM.mo
+share/locale/lt/LC_MESSAGES/Linux-PAM.mo
+share/locale/lv/LC_MESSAGES/Linux-PAM.mo
+share/locale/mk/LC_MESSAGES/Linux-PAM.mo
+share/locale/ml/LC_MESSAGES/Linux-PAM.mo
+share/locale/mn/LC_MESSAGES/Linux-PAM.mo
+share/locale/mr/LC_MESSAGES/Linux-PAM.mo
+share/locale/ms/LC_MESSAGES/Linux-PAM.mo
+share/locale/my/LC_MESSAGES/Linux-PAM.mo
+share/locale/nb/LC_MESSAGES/Linux-PAM.mo
+share/locale/ne/LC_MESSAGES/Linux-PAM.mo
+share/locale/nl/LC_MESSAGES/Linux-PAM.mo
+share/locale/nn/LC_MESSAGES/Linux-PAM.mo
+share/locale/or/LC_MESSAGES/Linux-PAM.mo
+share/locale/pa/LC_MESSAGES/Linux-PAM.mo
+share/locale/pl/LC_MESSAGES/Linux-PAM.mo
+share/locale/pt/LC_MESSAGES/Linux-PAM.mo
+share/locale/pt_BR/LC_MESSAGES/Linux-PAM.mo
+share/locale/ro/LC_MESSAGES/Linux-PAM.mo
+share/locale/ru/LC_MESSAGES/Linux-PAM.mo
+share/locale/si/LC_MESSAGES/Linux-PAM.mo
+share/locale/sk/LC_MESSAGES/Linux-PAM.mo
+share/locale/sl/LC_MESSAGES/Linux-PAM.mo
+share/locale/sq/LC_MESSAGES/Linux-PAM.mo
+share/locale/sr/LC_MESSAGES/Linux-PAM.mo
+share/locale/sr@latin/LC_MESSAGES/Linux-PAM.mo
+share/locale/sv/LC_MESSAGES/Linux-PAM.mo
+share/locale/ta/LC_MESSAGES/Linux-PAM.mo
+share/locale/te/LC_MESSAGES/Linux-PAM.mo
+share/locale/tg/LC_MESSAGES/Linux-PAM.mo
+share/locale/th/LC_MESSAGES/Linux-PAM.mo
+share/locale/tr/LC_MESSAGES/Linux-PAM.mo
+share/locale/uk/LC_MESSAGES/Linux-PAM.mo
+share/locale/ur/LC_MESSAGES/Linux-PAM.mo
+share/locale/vi/LC_MESSAGES/Linux-PAM.mo
+share/locale/yo/LC_MESSAGES/Linux-PAM.mo
+share/locale/zh_CN/LC_MESSAGES/Linux-PAM.mo
+share/locale/zh_HK/LC_MESSAGES/Linux-PAM.mo
+share/locale/zh_TW/LC_MESSAGES/Linux-PAM.mo
+share/locale/zu/LC_MESSAGES/Linux-PAM.mo
+share/man/man3/misc_conv.3
```

Comment:
Please remove documentation on man pages as there is no `man` command on the NAS

Reply:
Maybe a wiki entry is a better way to add documentation.

Reply:
Is removing the lines from PLIST enough or is there something to be added in the makefile?

Reply:
Yes the plist determines what is copied into the package